assets/app/scripts/services/data.js

```diff
@@ -150,6 +150,9 @@ angular.module('openshiftConsole')
     this._getNamespace(resource, context, opts).then(function(ns){
       $http(angular.extend({
         method: 'DELETE',
+        auth: {
```

Comment:
if we do `auth:{}`, that's a signal auth is required
anyone who doesn't want auth would do `{http:{auth:null}}`
anyone who wants custom auth options could do `{http:{auth:{opt:value}}}` and it wouldn't matter that we overrode `{}` set by data.js